Makefile

```diff
@@ -24,7 +24,7 @@ VALGRIND_OUTPUT_FAILURE_TAG := LEAK
 INCS := -I$(INCLUDE_TOP_DIR)
 TEST_INCS := -I/opt/gtest/include/
 
-CXX_FLAGS := $(INCS)
+CXX_FLAGS := -lncurses $(INCS)
```

Comment:
This flag is added to compilation of tests. At least for now this is unnecesary when compiling tests.

Reply:
Created bug #31